Onl 5094 vision file (#492)
Ebury/chameleon: VISION.md

# Vision

> Ebury Chameleon is developed thanks to the frontend expertise of the [Online team](https://fxsolutions.atlassian.net/wiki/spaces/TEAM/pages/119543461/ONL) members, Chameleon brings the components that Eburyonline needs to build the best online experience for the Ebury products.

Reviewer: Ebury labs link maybe? This Atlassian link won't work outside of the organisation

Reviewer 2: Minor thing, I prefer to keep this internal link.

Reviewer: fair enough

> The components from Chameleon are accessible through [https://chameleon.ebury.now.sh](https://chameleon.ebury.now.sh) and it's a live page when you can see all the components that the Online team has developed.

Reviewer 2: ```suggestion
The components from Chameleon are accessible through [https://chameleon.ebury.now.sh](https://chameleon.ebury.now.sh) and it's a live page where you can see all the components that the Online team has developed.
```

> This library is written on [Vue.js](https://vuejs.org/) and the UI component explorer is [Storybook](https://storybook.js.org), we follow the Design system [Chameleon](https://github.com/Ebury/chameleon) with the help of utility-first styles library [Tailwind CSS](http://tailwindcss.com).

Reviewer 2: ```suggestion
This library is written in [Vue.js](https://vuejs.org/) and the UI components explorer is [Storybook](https://storybook.js.org), we follow the design guidelines from the UX/Design team with the help of utility-first styles library [Tailwind CSS](http://tailwindcss.com).
```
